migrationmanagers/migrationmanagers.go

```diff
@@ -101,15 +101,15 @@ func (m *MigrationManager) ApplyToSchema(migrationRepo migrations.Repository) er
 	return nil
 }
 
-// Apply applies a migration to the TestTrack server and in-memory schema
-// without recording the version to TestTrack server
+// Apply idempotently applies a migration to the TestTrack server and in-memory
+// schema without recording the version to TestTrack server
 func (m *MigrationManager) Apply(migrationRepo migrations.Repository) error {
 	err := m.migration.Validate()
 	if err != nil {
 		return err
 	}
 
-	err = m.migration.ApplyToSchema(m.schema, migrationRepo)
+	err = m.migration.ApplyToSchema(m.schema, migrationRepo, true)
```

Comment:
Apply is used from `migrate` and `schema load`. There's an argument for both potentially not calling ApplyToSchema instead, but this is a competency I'd like ApplyToSchema to retain, and this will keep the codepath warm until we implement check.

Reply:
seems reasonable to me. i think that's the only argument for doing this idempotently flag. otherwise i'd say that we should change the code so that migrate doesn't update the schema since it happens in real-time in the other cases.